### PR TITLE
[PLAT-4029] Сleaning and optimization project

### PR DESCRIPTION
**Pull request by osa911**
[Link for task](https://reevoo.atlassian.net/browse/PLAT-4029)
- Deleted oldest dependencies
- Update all dependencies to lates version
- Optimized bundle file 179kb -> 128kb
- Added prettier and prettify code to company's style guide

### CODE REVIEW

File: lib/events/conversion.js [hunk 1]
```diff
@@ -2,73 +2,75 @@
...
-import stringify  from 'json-stable-stringify';
-import validate   from '../validate';
+import stringify from 'json-stable-stringify'
+import validate from '../validate'
 
-const EVENT_TYPES = ['purchase', 'checkout', 'propensity_to_buy'];
+const EVENT_TYPES = ['purchase', 'checkout', 'propensity_to_buy']
 
-function validationsForEventType(eventType) {
+function validationsForEventType(/* eventType */) {
```
Comment (robinbortlik) [inline lib/events/conversion.js]: Why this param is commented out? How the function works then?

File: lib/events/experiences.js [hunk 2]
```diff
@@ -2,24 +2,30 @@
...
 
 export default class Experiences {
   constructor(tracker) {
-    this.tracker = tracker;
+    this.tracker = tracker
   }
 
   dockToggled(opts) {
-    opts.fullDockToggledParams = JSON.stringify(opts);
+    // FIXME: should be like commented code below
```
Comment (robinbortlik) [inline lib/events/experiences.js]: Why you added FIXME and did not fix the code?

File: lib/events/conversion.js (hunk 1 above)
Comment (osa911) [reply lib/events/conversion.js]: This argument is not used in the function, as I understand it - it was left for future developers who will update this function.
But now, after update linter's rules it shows an error - rule: no-unused-vars, and I commented out this

File: lib/events/experiences.js (hunk 2 above)
Comment (osa911) [reply lib/events/experiences.js]: The first time i've fixed it - test was failed, and I returned it back but with FIXME.
As you can see in this func we mutate arguments, but it's a bad practice and i need to more investigate how it works before fix it. I've decided to do it but in another task. I think it's a correct. Do you agree with me?

**Pull request merged**